core/spec/lib/calculated_adjustments_spec.rb

```diff
@@ -120,13 +120,13 @@ class Calculable < ActiveRecord::Base
     end
 
     it "can update both calculator type and attributes" do
-      subject.update!(calculator_type: 'Spree::Calculator::FlexiRate', calculator_attributes: {
+      subject.update!(calculator_type: 'Spree::Calculator::Shipping::FlexiRate', calculator_attributes: {
         preferred_first_item: 123
       })
-      expect(subject.calculator.class).to eq(Spree::Calculator::FlexiRate)
+      expect(subject.calculator.class).to eq(Spree::Calculator::Shipping::FlexiRate)
       expect(subject.calculator.preferred_first_item).to eq(123)
       subject.reload
-      expect(subject.calculator.class).to eq(Spree::Calculator::FlexiRate)
+      expect(subject.calculator.class).to eq(Spree::Calculator::Shipping::FlexiRate)
```

Comment:
The non-namespaced calculators (except for `DefaultTax` and `FlatFee` which do taxes) are specific to the promotion system. I think it makes more sense to move them, because otherwise we end up with stuff in core that's really hard to test, because all of the proper collaborators are missing.